.pipelines/pipeline.yaml

```diff
@@ -281,7 +296,8 @@ stages:
           matrix:
             cni_dropgz:
               name: cni-dropgz
-              platforms: linux/amd64 linux/arm64
+              os_version: ltsc2019 ltsc2022
```

Comment:
if adding `os_version`, Makefile needs to be updated to make the dropgz/dropgz-test manifest targets match [this](https://github.com/Azure/azure-container-networking/blob/4c761480c152eb39c5b710327ebfb8e50da7c4ac/Makefile#L559)